docs/upgrade.md

```diff
@@ -0,0 +1,134 @@
+# From Docker Compose
+## Before you begin
+* [Install Tilt](quickstart.html) and Kubernetes if they are not yet installed.
+* If you're new to Tilt try stepping through [a Simple Tiltfile](first_config.html) first.
+
+## Differences between Docker Compose and Tilt
+* Docker Compose is configured with a static YAML config. Tilt is configured with a `Tiltfile`, written in a small subset of Python called
+[Starlark](https://github.com/bazelbuild/starlark#tour>).
+* Docker Compose uses runs services on Docker Machine or Docker Swarm. Tilt runs services natively on Kubernetes.
+
+## Migrate from Docker Compose to Tilt
+Let's take a simple `docker-compose.yml` file with one service:
+
+```yaml
+version: '3'
+services:
+  spoonerisms:
+    build:
+      context: ./spoonerisms
+      dockerfile: ./spoonerisms/Dockerfile
+    command: node /app/src/index.js"
+    volumes:
+      - ./spoonerisms:/app
+    ports:
+      - "9006:5000"
+```
+
+1. Create a `Tiltfile`
+2. Define your service name
+
+In Docker Compose your service name is a key in the services list:
+
+```yaml
+services:
+  spoonerisms:
+```
+
+In Tilt your service name is a function defined in your Tiltfile.
+
+```python
+def spoonerisms():
+
+```
+3. Set the build context
+
+In Docker Compose you can specify your build context and Dockerfile like so:
+
+```yaml
+services:
+  spoonerisms:
+    build:
+      context: ./spoonerisms
+      dockerfile: ./spoonerisms/Dockerfile
+```
+
+In Tilt you tell us where your Dockerfile is and we infer your build context from that.
+
+```python
+  img = static_build("spoonerisms/Dockerfile", "gcr.io/myproject/spoonerisms")
```

Comment:
should we talk about the `context` argument here?

Reply:
Since my docker compose config has `context` I think that's a good idea yeah.